python/tvm/relay/op/reduce.py

```diff
@@ -257,6 +257,38 @@ def mean(data, axis=None, keepdims=False, exclude=False):
     return _make.mean(data, axis, keepdims, exclude)
 
 
+def variance(data, axis=None, keepdims=False, exclude=False):
+    """Computes the variance of array elements over given axes.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        The input data
+
+    axis : None or int or tuple of int
+        Axis or axes along which a mean operation is performed.
+        The default, axis=None, will find the indices of minimum element all of the elements of
+        the input array. If axis is negative it counts from the last to the first axis.
+
+    keepdims : bool
+        If this is set to True, the axes which are reduced are left in the result as dimensions
+        with size one.
+        With this option, the result will broadcast correctly against the input array.
+
+    exclude : bool
+        If `exclude` is true, reduction will be performed on the axes that are
+        NOT in axis instead.
+
+    Returns
+    -------
+    result : relay.Expr
+        The computed result.
+    """
+    axis = [axis] if isinstance(axis, int) else axis
+    m = mean(data, axis, True, exclude)
+    return _make._variance(data, m, axis, keepdims, exclude)
+
+
```

Comment:
shall we add this wrap:
```
def moments(data, axis=None, keepdims=False):
    ....
    return mu, sigma
```
This will align with https://www.tensorflow.org/api_docs/python/tf/nn/moments

Reply:
Sure, I can add this. Is `moments` a generic name for this? Or should we just name it as something like `mean_variance`?

Reply:
Moments would mathematically be a worse name